modular_skyrat/modules/events/code/event_overrides.dm

```diff
@@ -125,3 +125,12 @@
  */
 /datum/round_event_control/alien_infestation
 	max_occurrences = 0
+	
+/**
+ * Paperwork Error
+ *
+ * Removed:
+ *Paperwork Error is too intrusive and should be staff-only.
+ */
+/datum/round_event_control/bureaucratic_error
+    max_occurrences = 0
```

Comment:
```suggestion
    max_occurrences = 0
    
```

Reply:
Not really needed, now there's two empty lines at the end of the file.